src/Microsoft.ML.Data/Transforms/ColumnConcatenatingTransformer.cs

```diff
@@ -212,22 +215,22 @@ internal ColumnInfo(ModelLoadContext ctx)
 
         private readonly ColumnInfo[] _columns;
 
-        public IReadOnlyCollection<ColumnInfo> Columns => _columns.AsReadOnly();
```

Comment:
>Columns [](start = 47, length = 7)

I believe this should be public too, since the users might want to inspect. 

@tomfinley

Reply:
Yes I agree, will fix.

---
In reply to: [253123508](https://github.com/dotnet/machinelearning/pull/2363#discussion_r253123508) [](ancestors = 253123508)